extensions/amp-form/0.1/amp-form.js

```diff
@@ -578,18 +573,55 @@ export class AmpForm {
     // Set ourselves to the SUBMITTING State
     this.setState_(FormState.SUBMITTING);
 
+    //Promised to run before all async calls ; require extended timeout
+    const increasedPresubmitPromises = [];
```

Comment:
Nit: Name is unclear about what's "increased". Suggestion:`requiredActionPromises`